tests/codestyle/PurgeWithoutDocCommentsTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
I think I would add this to [PurgeTest.php](https://github.com/blumilksoftware/codestyle/blob/main/tests/codestyle/PurgeTest.php)

Reply:
Why?
This is another test case where purge mode is tested but without removing the doc comments.

Reply:
But it's still the same fixer. You can run two different methods in one test class.

Reply:
aahh, okay